projects/scion/toolkit/viewport/src/viewport.component.spec.ts

```diff
@@ -718,10 +721,10 @@ describe('Viewport', () => {
     component.setStyle('viewport-content', {
       'height': '600px',
     });
-    fixture.detectChanges();
+    await flushChanges(fixture);
 
     expect(getSize(fixture, 'sci-viewport')).toEqual(jasmine.objectContaining({height: 300}));
-    expect(isScrollbarVisible(fixture, 'vertical')).toBeTrue();
+    expect(isScrollbarVisible(fixture, 'vertical')).toBe(true);
```

Comment:
Use `toBeTrue` since you don't provide an expectation fail output and to be consistent with the matcher below.